src/convertToObject.js

```diff
@@ -6,7 +6,36 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splitThis = sourceString.split(';');
```

Comment:
The variable name `splitThis` does not describe the contents of the variable well. A more descriptive name like `styleDeclarations` would increase code readability.